configs/chroma.yaml

```diff
@@ -12,7 +12,7 @@ llm:
     top_p: 1
     stream: false
 
-vectordb:
+vector_db:
```

Comment:
Hey @PatelKeviin, thanks for the contribution. We only want to change the yaml file structure from `vectordb` to `vector_db` and not inside the yaml file configuration. Can you please revert these changes in the yaml config?

Reply:
@deshraj thanks for your feedback. Please review my changes - I've reverted the yaml config file updates as advised, as well as reverted back the documentation reference updates to align.